modules/evvtMenu/evvtMenuUtils.php

```diff
@@ -106,6 +106,19 @@ function getMenuElements() {
 	return $menustructure;
 }
 
+function getsavedMenu() {
+	global $adb;
+	$sql = "SELECT savemenuid,menuname FROM `vtiger_savemenu`";
```

Comment:
where do you create the vtiger_savemenu table?  Don't you need to create a corebos updater changeset for that?